multiple-instances/rust/src/main.rs

```diff
@@ -52,6 +54,53 @@ async fn get_sdk(
     Ok(sdk)
 }
 
+/// On first run (if you don't already have a node), leave `mnemonic` as None. A new mnemonic is
+/// generated and printed to console.
+///
+/// On subsequent runs, or if you already have a node, set the `mnemonic`.
+///
+/// The device key `client-key.pem` and certificate `client.crt` are read from the current directory.
```

Comment:
I think there's a missing step in how to derive a client certificate from your partner certificate.
This is probably not the place to put that, but maybe a comment here? I'm not sure how that works tbh.